### ISSUE TITLE
[Bug] Incorrect fill_value handling in Torch backend's image.resize with pad_to_aspect_ratio=True

### ISSUE THREAD

**Issue by buildwithsuhana** [keras-team/keras#22902]
When using keras.ops.image.resize with pad_to_aspect_ratio=True and a custom fill_value in the PyTorch backend, the left-side (horizontal) padding is hardcoded to 1.0 (white) instead of using the provided fill_value. This leads to incorrect numerical results when padding is applied horizontally.

Colab notebook reproducing this issue: https://colab.research.google.com/drive/1zjNW3WEUS5N6yNXRogjiW4Abg7qt4vBW?usp=sharing

In JAX:
```python
img = np.zeros((1, 10, 10, 3), dtype="float32")
resized = keras.ops.image.resize(img, (10, 20), pad_to_aspect_ratio=True, fill_value=0.5)
res_np = keras.ops.convert_to_numpy(resized)
```

gives
```output
Left padding pixel (index 0):  [1. 1. 1.]
Right padding pixel (index 19): [0.5 0.5 0.5]
```

while other backends give:
```output
Left padding pixel (index 0):  [0.5 0.5 0.5]
Right padding pixel (index 19): [0.5 0.5 0.5]
```
which is the correct output as the fill value is set to 0.5 by the user.
